i18n/en.json

```diff
@@ -367,6 +367,14 @@
     "id": "api.channel.rename_channel.cant_rename_group_messages.app_error",
     "translation": "You cannot rename a group message channel"
   },
+  {
+    "id": "api.channel.undelete_channel.unarchived",
```

Comment:
```suggestion
    "id": "api.channel.restore_channel.unarchived",
```

Reply:
Proposing `restore_channel` to be consistent with the RestoreChannel API method

Reply:
done